lib/potassium/recipes/yarn.rb

```diff
@@ -0,0 +1,14 @@
+class Recipes::Yarn < Rails::AppBuilder
+  def create
+    template '../assets/package.json', 'package.json' unless get(:front_end)
+    template '../assets/bin/update.erb', 'bin/update', force: true
+    application "config.assets.paths << Rails.root.join('node_modules')"
+    append_to_file ".gitignore", "node_modules/\n"
+
+    if get(:heroku)
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.